resources/shared/test-runner.mjs

```diff
@@ -13,25 +13,52 @@ export class TestRunner {
         this.#test = test;
         this.#params = params;
         this.#callback = callback;
-
         this.#page = page;
         this.#frame = frame;
     }
 
-    async runTest() {
-        // Prepare all mark labels outside the measuring loop.
-        const suiteName = this.#suite.name;
-        const testName = this.#test.name;
-        const syncStartLabel = `${suiteName}.${testName}-start`;
-        const syncEndLabel = `${suiteName}.${testName}-sync-end`;
-        const asyncStartLabel = `${suiteName}.${testName}-async-start`;
-        const asyncEndLabel = `${suiteName}.${testName}-async-end`;
+    get page() {
+        return this.#page;
+    }
+
+    get test() {
+        return this.#test;
+    }
+
+    get syncStartLabel() {
+        return `${this.#suite.name}.${this.#test.name}-start`;
+    }
+
+    get syncEndLabel() {
+        return `${this.#suite.name}.${this.#test.name}-sync-end`;
+    }
+
+    get asyncStartLabel() {
```

Comment:
Do we really need to add all these getters now that we're only overriding _runSyncStep?

Reply:
I can revert this.

Reply:
done